common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -40,24 +41,25 @@ public Optional<Object> accept(Style style, String string) {
         return result.toString();
     }
 
-    public static String getUnformatted(Component msg) {
-        return msg.getString();
+    public static String getUnformattedString(FormattedText formattedText) {
+        // TODO: remove §-codes from string (when implemented don't forget to modify usages)
+        return formattedText.getString();
     }
 
     @Nullable
-    public static String getFormatted(String loreString) {
-        MutableComponent component = Component.Serializer.fromJson(loreString);
+    public static String getCodedString(String jsonString) {
+        MutableComponent component = Component.Serializer.fromJson(jsonString);
         if (component == null) return null;
 
-        return ComponentUtils.getFormatted(component);
+        return getCodedString(component);
     }
 
     @Nullable
-    public static String getUnformatted(String loreString) {
-        MutableComponent component = Component.Serializer.fromJson(loreString);
+    public static String getUnformattedString(String jsonString) {
```

Comment:
I'm seriously lost at this. It all seems just a mess to me. It definitely breaks the "don't make me think" rule. I can accept it for now, if it is needed. But we definitely need to find a better way to handle strings to make it unambiguous.

Reply:
("this" meaning all these changes to this file)